CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 6.9.6 - 2022/03/31
```

Comment:
I think the `VERSION` and `Gemfile`/``.lock` will need updating?

Reply:
Hmm, VERSION updated but the Gemfile didn't